builder/layouts/screenshot.hbs

```diff
@@ -1,21 +1,28 @@
-{{> head }}
-
-<style>
+<!doctype html>
+<html lang="en">
+<head>
+  {{> head }}
+  <title>{{ title }}</title>
+  <style>
     .mdl-card__menu img {
-        height: 86px;
-        margin-top: -20px;
-        margin-right: -30px;
+      height: 86px;
+      margin-top: -20px;
+      margin-right: -30px;
     }
-</style>
+  </style>
+</head>
 
-<div class="mdl-card mdl-shadow--2dp card snippet-card">
+<body>
+  <div class="mdl-card mdl-shadow--2dp card snippet-card">
     <div class="mdl-card__title">
-        <h2 class="mdl-card__title-text">✅ {{title}} </h2>
+      <h2 class="mdl-card__title-text">✅ {{title}} </h2>
     </div>
     <div class="mdl-card__supporting-text">
-        {{{markdown content}}}
+      {{{markdown content}}}
     </div>
     <div class="mdl-card__menu">
-        <img src="public/logo.png" style="">
+      <img src="public/img/logo.png" style="" alt="">
     </div>
-</div>
+  </div>
+</body>
```

Comment:
Also move to a partial?

Reply:
this will require 3 partials at least (opening head, closing head/opening body, closing body) and markup will be broken in these partials which I don't like. 
So, having html/body in layout is more explicit and clear rather than having this into partials.